### ISSUE TITLE
Correct heyMP email in git history

### ISSUE THREAD

**Issue by heyMP** [elmsln/elmsln#474]
I was wondering why I've been not existent in Github.  *facepalm*

Changing Email Addresses Globally
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#The-Nuclear-Option:-filter-branch

**Comment by btopro** [comment]
tagging this as everything to be ridiculous since this should have been done awhile ago and requires like 30 minutes of effort ;)

**Comment by btopro** [comment]
I think you should fix this

**Comment by mmilutinovic1313** [comment]
On it. k thnx!
